core/src/main/java/org/elasticsearch/indices/IndexAlreadyExistsException.java

```diff
@@ -32,7 +32,7 @@
 public class IndexAlreadyExistsException extends ElasticsearchException {
 
     public IndexAlreadyExistsException(Index index) {
-        this(index, "already exists");
+        this(index, "index " + index.getName() + " already exists");
```

Comment:
Would you mind changing this to `index#toString` so that it includes the index UUID as well?